tests/contrib/pymongo/test.py

```diff
@@ -572,8 +572,8 @@ def check_socket_metadata(span):
         assert span.name == "pymongo.get_socket"
         assert span.service == mongox.SERVICE
         assert span.span_type == SpanTypes.MONGODB
-        assert span.meta.get("out.host") == "localhost"
-        assert span.metrics.get("out.port") == MONGO_CONFIG["port"]
+        assert span.get_tag("out.host") == "localhost"
+        assert span.get_metric("out.port") == MONGO_CONFIG["port"]
```

Comment:
this shouldn't be here right? this should have already been updated? or was this missed before?

Reply:
I think it got mixed up in a rebase [😅](https://github.com/DataDog/dd-trace-py/pull/3201/files#r804086547). It was a messy change. Good catch

Reply:
Yep, looks like because of the rebase where #2526 has overwritten the change on master. This will either get fixed by this PR or when we do the next rebase of `next-1.0-dev`.